uptane/encoding/asn1_definitions.py

```diff
@@ -589,7 +589,7 @@ class VehicleVersionManifestSigned(univ.Sequence):
     namedtype.NamedType('vehicleIdentifier', Identifier().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 0))),
     namedtype.NamedType('primaryIdentifier', Identifier().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 1))),
     namedtype.NamedType('numberOfECUVersionManifests', Length().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 2))),
-    namedtype.NamedType('ecuVersionManifests', ECUVersionManifests().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 3))),
+    namedtype.NamedType('ecuVersionManifests', ECUVersionManifests().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 3))), # Should this be tagFormatConstructed?
```

Comment:
Oh. Right. This is dynamically generated. So I guess the question is: why is it being generated with tagFormatSimple rather than tagFormatConstructed?

(I should look into this further when there's time.)